src/metadata/metacontent_handler.h

```diff
@@ -47,8 +47,8 @@ class MetacontentHandler : public MetadataHandler {
 class FanArtHandler : public MetacontentHandler {
 public:
     explicit FanArtHandler(const std::shared_ptr<Context>& context);
-    void fillMetadata(std::shared_ptr<CdsObject> obj) override;
-    std::unique_ptr<IOHandler> serveContent(std::shared_ptr<CdsObject> obj, int resNum) override;
+    void fillMetadata(std::shared_ptr<CdsObject> item) override;
+    std::unique_ptr<IOHandler> serveContent(std::shared_ptr<CdsObject> item, int resNum) override;
 
```

Comment:
It's `CdsObject `so the paramter should be named `obj`. Maybe it's not done in MetadataHnadler

Reply:
Please rename the arguments in implentation. Declaration is as it should be.